### DISCUSSION TITLE
Let the screenshot in the Preview Image capture the HTML UI

### DISCUSSION THREAD

**Discussion by karlosgliberal** [cables-gl/cables_docs#386] [Feature Requests] [answered]
Hello, first of all thank you very much for all the work and for the amount of continuous improvements.

It would be cool if the preview Image also captures the UIs that are created with Html Ops like DivElemnts, CssOp.
I put a screenshot as an example:

Thanks for everything
<img width="1792" alt="screenshot" src="https://user-images.githubusercontent.com/200922/198671797-4f2d47b0-90a6-4d68-8f5b-a7dcfad771af.png">

**Comment by steam0r** [comment]
that sadly is not possible as we cannot screenshot html elements, only the rendered canvas. sorry :/

**Comment by pandrr** [comment, marked as answer]
yes we would love that, sadly its not possible to capture html elements to pixels/webgl textures.

here is a horrible hack: 

1. make screenshot manually
2. add fullscreenrectangle
3. attach screenshot as texture to fullscreenrectangle
4. make manual screenshots in project settings and turn off auto screenshot

this is how we make screenshots vor viz ops...

**Reply by karlosgliberal** [reply]
Thanks, I understand.
It is true that in inspection the html is above the canvas.

But I like the hack ;) I'll use it for the next time
